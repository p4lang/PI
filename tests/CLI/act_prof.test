#!/bin/sh
./run_one_test.py $srcdir/testdata act_prof ecmp.json
