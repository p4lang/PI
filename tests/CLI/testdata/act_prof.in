act_prof_create_member ecmp_action_profile set_nhop 10.0.0.1 7
act_prof_create_group ecmp_action_profile
act_prof_add_member_to_group ecmp_action_profile 0 9223372036854775808
act_prof_dump ecmp_action_profile
