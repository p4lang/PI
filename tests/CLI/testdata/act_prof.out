????
Device assigned successfully.
Selecting device.
????
Member was successfully created with handle 0.
????
Group was successfully created with handle 9223372036854775808.
????
Member was successfully added to group.
????
Successfully retrieved 1 member(s) and 1 group(s).
==========
MEMBERS
**********
Dumping member 0
Action entry: set_nhop - 0a000001, 0007
==========
GROUPS
**********
Dumping group 9223372036854775808
Members: [0]
????
